src/core/cuda/cuda_profile.cc

```diff
@@ -116,7 +116,11 @@ void CUPTIAPI bufferCompleted(
   // since we launched only 1 kernel, we should have only 1 kernel record
   TC_CUPTI_CHECK(cuptiActivityGetNextRecord(buffer, validSize, &record));
 
+#if (CUPTI_API_VERSION >= 10)
```

Comment:
This test doesn't seem to match the commit message.
You're doing something special for versions "> 9.0" not "< 9.0".
I don't know which of the two is correct.
However, you seem to be modifying new code, so this should be squashed in.
There is no point in introducing known broken commits.

Reply:
The commit message refers to CUDA version. The check is on CUPTI version (their versions are tied but they use different numbers).

Reply:
I'll squash it in the right commit.